### ISSUE TITLE
python: ucs2 vs ucs4  

### ISSUE THREAD

**Issue by rmarabini** [I2PC/scipion#397]
when trying to debug using eclipse/gdb I got estrange menages. In particular

gdb --version returns:
   gdb: symbol lookup error: gdb: undefined symbol: PyUnicodeUCS4_FromEncodedObject

Scipion python is compiled with ucs2 on while most of the operative system have it set to ucs4.
Users occasionally get binaries built for a compatible Linux and Python version but with a different UCS2-vs-UCS4 (gdb was compiled for ucs4) setting,and those users get mysterious memory corruption errors which are hard to diagnose. 

In summary I suggest to set scipion-python to compile by defect with ucs4. This is achieved by editing file install/script.py:    and setting 

flags=['--enable-shared','--enable-unicode=ucs4'],

**Comment by azazellochg** [comment]
I have the same error after installing devel version from scratch:
```
ImportError: /usr/local/scipion/software/lib/python2.7/site-packages/matplotlib-1.3.1-py2.7-linux-x86_64.egg/matplotlib/ft2font.so: undefined symbol: PyUnicodeUCS4_GetSize
```
I will try to modify flags for python as suggested.

**Comment by azazellochg** [comment]
This is no longer relevant for python3: https://docs.python.org/3.8/c-api/unicode.html
